FileManager/app/src/main/res/xml/provider_paths.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please add the Apache 2.0 license header that's found in the other files. :)

Reply:
Added here 683451ad760cdcbd17b158358a0ec27f267e4f07